### ISSUE TITLE
📢dotnet/runtime schedule for the remainder of .NET 7

### ISSUE THREAD

**Issue by danmoseley** [dotnet/runtime#72297]
It's about 4 months until .NET 7 ships ("GA") in November. dotnet/runtime will be following the same procedure that we did for .NET 6, as follows -- 

* 12-July -- we branched main for Preview 7 and the team has now changed focus from feature development to open bug fixing for .NET 7
* 15-Aug -- we branch main for RC1. After this point, main will be a .NET 8 branch. We will selectively backport fixes to the .NET 7 release branch as needed, with a process similar to the one we use for servicing (but a little more freely). Many or most changes in main will not be backported. No features will be backported.
* 13-Sept -- we branch our release branch for RC2. Nothing changes for those working in main. The bar for changes to be ported into .NET 7 becomes considerably higher (centralized permission process of taking to .NET tactics meeting)
* 10-Oct -- last ship stopper fix merged if any.

Where do community changes go? main remains open at all times for community changes, even adding new approved API. Many will get into .NET 7 if they go in before 15-Aug. However if a change brings too much risk we will hold merging it until 16-Aug.

TLDR: if you plan to work on a change that you want in the .NET 7 product, 15-Aug is the date it needs to be merged by.

Nearer the fall, we'll start thinking and planning for .NET 8, and we'll be sharing our thinking when we do.

**Comment by danmoseley** [comment]
During this process we are monitoring issues by area that have the .NET 7 milestone, much as we did for .NET 6. Below is a snapshot from earlier in the week. We will be taking each issue and either fixing it, closing it, or setting to the Future milestone (meaning discussion is worth continuing, or we would accept a change for it, but we haven't made a commitment to make the change ourselves).

![image](https://user-images.githubusercontent.com/6385855/179321749-dec2ab9b-a8ba-45f8-b0a8-b732f9875f2b.png)

"ZBB" in the diagram is an old Microsoft term (zero bug bounce) meaning the point where we aim to get zero bugs in the milestone. It's called a bounce as inevitably some must fix bugs are discovered after that point.

**Comment by danmoseley** [comment]
How are we choosing what to fix? We use criteria like the following to decide what to fix in the remaining time:

1. Broken or significantly incomplete 7.0 scenario
1. Resolves significant customer feedback on 7.0 feature
1. High impact regressions from last release
1. Significant reliability issues
1. Unapproved significant compatibility breaks
1. Breaking PR validation or infrastructure
1. Significant unapproved performance regression from last release
1. Anything we would service for such as security.

As we go along the criteria become increasingly stringent in order to minimize the risk of introducing new issues.

Issues that can't be fixed will be closed or set to Future for further discussion or future work.

**Comment by filipnavara** [comment]
Why was this unpinned?

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/179528925-65c32c27-58e9-4dcf-970d-8b354cb8475d.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/180797395-2f051553-67a7-4f3a-b091-8f17a2c7d22a.png)

**Comment by danmoseley** [comment]
@VincentBu  please leave pinned

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/182181636-7a7c1cb3-e287-4cb3-81a4-a4e042ec5515.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/183447178-a7189bf6-deab-4c40-baac-7d07de3ed820.png)

**Comment by danmoseley** [comment]
@jeffschwMSFT not sure how to get this issue itself out of your stats.

**Comment by jeffschwMSFT** [comment]
:) there is not a way, it will just be part of our non-product issues.  We have a lot of samples, docs, etc. that we track towards the end of the release that hang around.

**Comment by danmoseley** [comment]
On 5pm PDT on Monday 15th, main will become a .NET 8 branch. Here is where we are at:

![image](https://user-images.githubusercontent.com/6385855/184466487-a600c2da-f8fd-4931-89ad-6cb23907c3bb.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/184653915-07ec990f-2be2-40ee-b89e-099f1cfb4829.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/185944618-c26fdee3-2713-4edc-b361-05d6779b37a1.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/187233033-a65822be-1e47-44bf-968b-1419ef70915f.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/188478699-c78c0228-0416-43b5-a6ef-03d39b303cf8.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/189675268-0b78a61a-c056-432f-9403-0a5a858b8f6a.png)

**Comment by danmoseley** [comment]
Linking issuesof query for the ~50 remaining (although someone said Github issue search allows "or" now?)
https://issuesof.net/?q=is%3Aopen%20is%3Aissue%20repo%3Aruntime%20%28milestone%3A7.0.0%20or%20label%3Ablocking-release%29

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/190176731-5ee12865-f4de-4356-a494-442939a35864.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/191037612-a5bb84e0-67a2-4ad5-8722-3adb6aec528b.png)

**Comment by jeffschwMSFT** [comment]
![image](https://user-images.githubusercontent.com/9561583/192313290-fe819056-1a77-47ec-bd15-e7dc7f2026a5.png)

Last update for .NET 7.  On our way to GA
